pkgs/by-name/fl/flatpak/package.nix

```
@@ -266,5 +266,6 @@ stdenv.mkDerivation (finalAttrs: {
     license = lib.licenses.lgpl21Plus;
     maintainers = with lib.maintainers; [ getchoo ];
     platforms = lib.platforms.linux;
+    mainProgram = "flatpak";
```

```suggestion
    mainProgram = "flatpak";
    platforms = lib.platforms.linux;
```
Nitpick about the ordering :p

---

Fixed, no problem. Just curious on what the order should be so I know in the future.

---

There's not really a standard; I just personally follow the order they're listed in the [Meta-attributes chapter](https://nixos.org/manual/nixpkgs/unstable/#chap-meta) of the manual